### PR TITLE
Updated README, step-by-step installation procedure

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,17 @@ This cluster operator gathers anonymized system configuration and reports it to
 
 ## Building
 
-To build the operator, install Go 1.11 or above and run:
+To build the operator, install Go 1.11 or above. Don't forget to set `$GOPATH` environment variable properly.
+
+Then retrieve sources of insights-operator:
+
+    go get -u github.com/openshift/insights-operator
```
Comment (mfojtik) [inline README.md]: This is not very common way to get the sources ;-) We usually do `git clone`.

Comment (natiiix) [reply README.md]: Wouldn't this make importing more convenient?

**Comment by tisnik** [comment]
@mfojtik I'd change the 'go get' to the following command
'git clone https://github.com/openshift/insights-operator $GOPATH/src/github.com/openshift/insights-operator'

Would it be better?

**Comment by tisnik** [comment]
Finally we updated that repo to use Go modules, so this PR is no longer relevant :)

**Pull request closed without merging**